src/schema/resolvers/invitation/mutation.resolver.js

```diff
@@ -0,0 +1,122 @@
+import { invitationTriggers } from "utils/pubSubTriggers";
+import { fromInvitationDtoToInvitation } from "utils/mapper/invitation";
+
+const createInvitation = async (_, args, context) => {
+  // get Event for invitation
+  const { data: eventData } = await context.database.events.getById(
+    args.eventId
+  );
+
+  // calculate random 6 digit generated code
+  const validCode = false;
+  const { data: invitations } = await context.database.invitations.find({
+    eventId: args.eventId,
+  });
+  let randomCode = "";
+  while (!validCode) {
+    const code = Math.floor(100000 + Math.random() * 900000).toString();
+    const checkExistingCode = invitations.filter(
+      (invitation) => invitation.code === code
+    );
+    if (checkExistingCode.length === 0) {
+      randomCode = code;
+      break;
+    }
+  }
+
+  const currentDateTime = new Date().toLocaleString();
```

Comment:
use UTC string date format. The value of `toLocaleString` ends up using the local date of where the server is deployed (which is somewhere in america). May get timezone issues

Reply:
hmm should we convert to GMT on the BE then?

Reply:
Yeah I think so. Same thing goes when sending the date from FE to BE